code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -53,7 +53,7 @@
 
 	//primitive_form = "Farwa"
 
-	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED | SPECIES_WHITELIST_SELECTABLE//Whitelisted as restricted is broken.
+	spawn_flags = SPECIES_CAN_JOIN //Whitelisted as restricted is broken.
```

Comment:
You can define this var in a file within a file inside the `.../content_arfs` folderand it will override any code inside the `../code` folder.

For simple variable definitions, this is preferred to editing the base files